configuration/scripts/forapps/ufs/comp_ice.backend.clean

```diff
@@ -10,10 +10,10 @@ setenv OBJDIR $EXEDIR/compile           ; if !(-d $OBJDIR) mkdir -p $OBJDIR
 
 if (${SITE} =~ cheyenne*) then
   setenv ARCH cheyenne_intel
-#else if (${SITE} =~ Orion*) then
-#  setenv ARCH orion_intel
-#else if (${SITE} =~ hera*) then
-#  setenv ARCH hera_intel
+else if (${SITE} =~ Orion*) then
```

Comment:
Did you try this on orion? I had made a change so that it would look for "orion" and not "Orion".

Reply:
I have not tried it on Orion.

Reply:
I am trying this fix along w/ the debug fix on orion. I think once we make the change above for the clean (from Orion to orion) this will ready.

Reply:
Are you testing this on Orion? If not, I can test it.

Reply:
I am trying. I'm getting this error in:
CICE6 ./comp_ice.backend.libcice: ARCH = orion_intel
ICE_BLDDEBUG: Undefined variable.

Reply:
I think I see my error. Let me try again.

Reply:
I've tested both debug and normal compile on hera and orion. @MinsukJi-NOAA could you push the fix to the backend.clean to change Orion to orion? Then I will approve. Thanks!

Reply:
Thanks for testing this!